.gitignore

```diff
@@ -1,6 +1,7 @@
 .build
 node_modules
 .DS_Store
+.idea
```

Comment:
I added this folder to .gitignore as I use Webstorm. Should I add .vscode too?

Reply:
👍 good idea

Reply:
Done!